tests/transfero_test/synonyms_test.py

```
@@ -4,12 +4,132 @@
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../..")))
 
 import pytest
+from src.core.accido.endings import Adjective, Noun, Verb
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)